### PR TITLE
Cope with multiple committee chairings.

### PR DESCRIPTION
**Pull request by dracos**

### CODE REVIEW

File: scripts/datadotparl/update-members [hunk 1]
```diff
@@ -71,6 +71,13 @@ class Position(object):
     self.position = position
     self.list = positions
 
+  def chair_check(self, chair_date, periods, to_date):
+    if chair_date[1] < to_date:
+      periods.append({'position': 'Chair', 'fromdate': chair_date[0], 'todate': chair_date[1]})
+      periods.append({'position': '', 'fromdate': chair_date[1], 'todate': to_date})
+    else:
+      periods.append({'position': 'Chair', 'fromdate': chair_date[0], 'todate': to_date})
+
```
Comment (struan) [inline scripts/datadotparl/update-members]: I take it from this that we are assuming that a Chair is in place for a single stretch and any gaps are due to the dissolution of Parliament for elections?

**Comment by struan** [comment]
One question but assuming that the answer to that is yes, looks good.

File: scripts/datadotparl/update-members (hunk 1 above)
Comment (dracos) [inline scripts/datadotparl/update-members]: The only entry with two chair dates at present is: Member for a week, Chair for a year, Member for a few months (yes, this presumably was over election period, but doesn't have to be), Chair until now. The code should handle this correctly with the membership in-between the two chairs?

Comment (struan) [inline scripts/datadotparl/update-members]: Yup, understand it now.

**Pull request merged**